.github/workflows/compliance.yml

```diff
@@ -50,8 +50,7 @@ jobs:
         # debug
         git log  --pretty=oneline | head -n 10
         west init -l .
-        west update
-        west zephyr-export
```

Comment:
what is this used for?  not required anymore?

Reply:
I dont know why we did `west zephyr-export` inside the compliance workflow. It is only used to register current zephyr installation as a cmake configuration package. And we dont 'build' any samples in compliance workflow. Essentially rendering this useless.

Reply:
Agreed